scalyr_agent/util.py

```diff
@@ -105,7 +106,7 @@ def json_dumps_custom(obj, fp):
             :param fp: If not None, then a file-like object to which the serialized JSON will be written.
             :type obj: dict
             :return: If fp is not None, then the string representing the serialization.
-            :rtype: Python3 - six.text_type, Python2 - six.text_type | six.binary_type
+            :rtype: Python3 - six.text_type, Python2 - six.binary_type
```

Comment:
Add similar comment to above at line 104 here.

Reply:
Done.